### PR TITLE
Map text translation 25/12/2024

### PR DESCRIPTION
**Pull request by allensw00**

### CODE REVIEW

File: Map-Message/ze_boralus.json [hunk 1]
```diff
@@ -87,17 +87,17 @@
...
   },
   "正在清理障碍, 30 seconds left": {
-    "Translation": "正在清理障碍, 30 seconds left",
+    "Translation": "Clearing obstacles, 30 seconds left",
     "M-Bool": false,
     "Value": 0
   },
   "这儿是伯拉勒斯的汇帆市场": {
-    "Translation": "这儿是伯拉勒斯的汇帆市场",
+    "Translation": "Here is the Belaruss Exchange Market",
```
Comment (Oz-Lin) [inline Map-Message/ze_boralus.json]: Belaruss -> Bolarus

File: Map-Message/ze_boralus.json [hunk 2]
```diff
@@ -112,22 +112,22 @@
     "Value": 0
   },
   "前方是戴姆维克街区": {
-    "Translation": "前方是戴姆维克街区",
+    "Translation": "Ahead is the Demevic District",
```
Comment (Oz-Lin) [inline Map-Message/ze_boralus.json]: Demevic District -> Dampwick Ward

File: Map-Message/ze_boralus.json [hunk 3]
```diff
@@ -177,33 +177,33 @@
...
   },
   "stage 3 complete": {
-    "Translation": "stage 3 complete",
+    "Translation": "stage 3 complete. EN Translation: AllenW @ Ozlin Server",
     "M-Bool": false,
     "Value": 0
   },
   "回到汇帆市场": {
-    "Translation": "回到汇帆市场",
+    "Translation": "Return to the Exchange Market",
```
Comment (Oz-Lin) [inline Map-Message/ze_boralus.json]: Exchange Market -> Tradewinds Market

**Review by Oz-Lin — CHANGES_REQUESTED** [review]

**Comment by allensw00** [comment]
Updated

**Pull request merged**